src/components/AddView.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../styles/addView.css';
+
+const AddView = () => {
+  return (
+    <div className="add-view">
+      <ul>
+        <li>
+          <Link to="/list-view">List View</Link>
+        </li>
+        <li className="add">
+          <Link to="/add-view">Add View</Link>
+        </li>
```

Comment:
Everything works well. I just have one suggestion that you can take into consideration or not 😊

react-router-dom has a special version of the `<Link>` tag called `<NavLink` that helps you to add styling attributes. This means that you can create selected routes boldly easier. You can read more about it [here](https://reactrouter.com/web/api/NavLink/activeclassname-string).

Reply:
I think I have applied the changes, the way its needed